app/views/new.html.erb

```diff
@@ -0,0 +1,19 @@
+<%= provide :title, t("sign_up") %>
```

Comment:
```suggestion
<%= provide :title,"Sign Up") %>
```
If you don't use i18n

Reply:
i already put sign up in en.yml & my.yml